packages/core/admin/admin/src/components/NpsSurvey.tsx

```diff
@@ -189,6 +189,7 @@ const NpsSurvey = () => {
         environment: currentEnvironment,
         version: strapiVersion ?? undefined,
         license: window.strapi.projectType,
+        onCloud: process.env.STRAPI_HOSTING === 'strapi.cloud'
```

Comment:
Should we use `isHostedOnStrapiCloud` to match the name [here](https://github.com/strapi/strapi/blob/develop/packages/cli/create-strapi-app/src/utils/usage.ts#L40)?

Reply:
very nit picky but I prefer isXYZ as a naming convention for booleans like this, happy to go with this though

Reply:
well, if it's two of you calling for it, I should change it

Reply:
done